doc/data/messages/b/bad-mcs-method-argument/bad.py

```diff
@@ -0,0 +1,3 @@
+class Meta(type):
+    def __new__(some):
+        pass
```

Comment:
There need to be a message raised by the bad exampele somewhere, you can launch the check with ``pytest doc -k bad-mcs``

Reply:
I found a message for the code

Reply:
I made an issue happen